src/OagBundle/Resources/views/Default/index.html.twig

```diff
@@ -54,7 +54,7 @@
         </div>
     </ul>
 
-    <h2>Enhancement Documents</h2>
+    <h2>Enhancement Documents <small><small>[activities/locations]</small></small></h2>
```

Comment:
Why are we nesting two small tags here?

Reply:
To go even smaller ;)

![image](https://user-images.githubusercontent.com/2150762/30066218-255d49be-924f-11e7-9b7d-abbe11ec4635.png)

Reply:
Whoa